### DISCUSSION TITLE
Screen is flashing

### DISCUSSION THREAD

**Discussion by ayyappa-flix** [google/model-viewer#4621] [Q&A]
Hi,

while loading the model. I can see like Screen is flashing here is the model example
https://media.flixcar.com/3dfw/p/player/index.html?arid=1825115
can you help me with this @elalish
thanks

**Comment by danilokrulj** [comment]
![image](https://github.com/google/model-viewer/assets/92090684/f6b4a85a-d453-431b-82bf-271f4f5c18ab)
![image](https://github.com/google/model-viewer/assets/92090684/9e7b6de2-2e55-4c9d-9045-aeb6a6132ca3)

If you are talking about this issue with the TV screen I think you just need to move the "picture" plane a little bit away from the frame of the TV.

**Reply by ayyappa-flix** [reply]
Hi, @danilokrulj 

Yes, I want to remove the black shade on the screen. is there any way to fix this kind of model please suggest.

thanks

**Reply by danilokrulj** [reply]
You just need to move the plane that has the screen texture a bit further away from the rest of the model or the frame, the plane and the object behind it are in the same position so there is z-fighting as elalish explained, basically the planes are overlapping and the renderer cant really decide what is in front.

**Comment by elalish** [comment]
Agreed, that looks like z-fighting. Ideally models like this should be authored as a single shiny, textured surface. Putting a glass layer over the image might sound more physically realistic, but in practice it only makes things worse.

**Reply by ayyappa-flix** [reply]
Hi, @elalish 

Yes, I want to remove the black flashing on the screen. is there any way to fix this kind of model please suggest.

thanks
